tests/ops/api/v1/endpoints/test_user_endpoints.py

```diff
@@ -340,10 +355,6 @@ def test_delete_user_as_root(self, api_client, db, generate_auth_header, user):
         )
         assert permissions_search is None
 
-        # Deleted user's client is also deleted
-        client_search = ClientDetail.get_by(db, field="id", value=client_id)
-        assert client_search is None
```

Comment:
If this test was failing here that looks like a bug — we still want the deleted user's client removed.

Reply:
Confused about what these lines do compared to the exact same thing a few lines above.

Reply:
Apologies — the diff didn't show that the code you removed was duplicate.